### ISSUE TITLE
Is is possible for a local notification to be permanent?

### ISSUE THREAD

**Issue by icecandy** [MaikuB/flutter_local_notifications#1524]
I would like to create a persistent local notification on Android. One that stays on screen until the user clicks it. Upon clicking an action is called and the notification is dismissed.

If I use the following with ongoing = true and autoCancel=false, I expected it to do this, but it doesn't - the notification disappears automatically after about 5 or 6 seconds. Am I completely misinterpreting what is supposed to happen here? Also, does timeoutAfter have any bearing on this (it doesn't seem to make any difference at all):
```
flutterLocalNotificationsPlugin.show(
          id,
          title,
          body,
          NotificationDetails(
            android: AndroidNotificationDetails(
              channel.id,
              channel.name,
              channelDescription: channel.description,
              importance: Importance.max,
              priority: Priority.high,
              autoCancel: false,
              //timeoutAfter: 1000000,
              ongoing: true,
            ),
          ),
        );
```

**Comment by MaikuB** [comment]
> Am I completely misinterpreting what is supposed to happen here? 

Possibly. Using `ongoing` will keep the notification. However, I do wonder if you've also confused in that when you said it disappears that the notification is actually still there in the status bar. This can reproduced in the example app and is the expected behaviour from Android. Were you perhaps expecting that the [heads-up notification](https://developer.android.com/guide/topics/ui/notifiers/notifications#Heads-up) would remain? If so then I believe that's not possible.

> Also, does timeoutAfter have any bearing on this (it doesn't seem to make any difference at all):

In your code it's commented out so no and shouldn't be used in your scenario either

**Comment by icecandy** [comment]
@MaikuB yes I was trying to create a situation where the notification remains on screen. You say it's not possible and it seems this is the standard Android approach (different to iOS where you can have a persistent notification alert on screen). I just wanted to confirm this so thanks very much. It is trivial to create one's own custom local notification UI, but then you don't get queueing etc. My notifications are very infrequent so that isn't a problem.
PS my question about timeoutAfter obviously means if it is used and not commented out! I can't imagine anyone thinking that commented out code does anything!

**Comment by MaikuB** [comment]
I'll close this issue then since the question has been answered

Note i called out the commented code as sometimes developers, including myself, think something isn't working but had missed something like this. Furthermore when I tried to include that property in combination with ongoing, the notification didn't even fire with no errors reported. Not sure if there's an issue with the Android SDK
